spec/factories/miq_alert_status_action.rb

```diff
@@ -0,0 +1,6 @@
+FactoryGirl.define do
+  factory :miq_alert_status_action do
+    action_type 'comment'
+    comment     'https://www.youtube.com/watch?v=dQw4w9WgXcQ'
```

Comment:
I'm disappointed @chrisarcand and @Fryguy didn't see this.  👍 @moolitayer

Reply:
I showed this to you because I was *literally, perfectly* rick-rolled looking through commits working on something else. 😅

Reply:
:bowtie:
The SOP shows up on the monitoring screen (it's supposed to help you solve the alert)
and we had a couple of rick rolling demos of it which was too much so I removed it from setup examples...